torchok/data/datasets/base.py

```diff
@@ -79,6 +93,11 @@ def _read_image(self, image_path: str) -> np.ndarray:
                 image = cv2.cvtColor(image.astype('uint8'), cv2.COLOR_RGB2BGR)
             elif image.shape[2] == 3:  # RGB
                 image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
+            elif image.shape[2] == 2:  # Gray with Alpha, LA mode in Pillow
+                gray = image[..., 0]
+                alpha = image[..., 1] / 255
+                image = (gray * alpha + self.rgba_layout_color * (1 - alpha)).astype(np.uint8)
```

Comment:
Add `np.clip` and move this line after `cv2.cvtColor` because self.rgba_layout_color mat be a list of 3 items